src/main/kotlin/com/sourcegraph/cody/error/CodyErrorSubmitter.kt

```diff
@@ -46,13 +47,22 @@ class CodyErrorSubmitter : ErrorReportSubmitter() {
       throwableText: String? = null,
       additionalInfo: String? = null
   ): String {
-    return "https://github.com/sourcegraph/jetbrains/issues/new" +
-        "?template=bug_report.yml" +
-        "&labels=bug" +
-        "&projects=sourcegraph/381" +
-        (throwableText?.let { "&title=${encode(getTitle(throwableText))}" } ?: "") +
-        "&about=${encode(getAboutText(project).get())}" +
-        "&logs=${encode(formatLogs(throwableText, additionalInfo))}"
+    val baseUrl =
+        "https://github.com/sourcegraph/jetbrains/issues/new" +
+            "?template=bug_report.yml" +
+            "&labels=bug" +
+            "&projects=sourcegraph/381"
+
+    val title = throwableText?.let { "&title=${encode(getTitle(it))}" } ?: ""
+    val text = getAboutText(project).get()
+    val about = "&about=${encode(text.repeat(6))}"
+
+    val availableSpace =
```

Comment:
It is impossible to be negative? :)

Reply:
it is but it would require to have probably over 500 plugin installed (so that you IDE Information gets really big)